src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Theme/index.html.twig

```diff
@@ -40,187 +40,123 @@
+            </p>
           </div>
-          <div class="card-footer">
-            <div class="d-flex justify-content-end">
-              <button class="btn btn-primary">
-                <i class="material-icons">save</i>
+        {% endif %}
+
+        {% if isInstalledRtlLanguage %}
+            {% include '@PrestaShop/Admin/Improve/Design/Theme/Blocks/rtl_configuration.html.twig' %}
+        {% endif %}
+
+        {{ form_start(shopLogosForm.options, {'action': path('admin_themes_upload_logos')}) }}
+          {% if not isInstalledRtlLanguage and isSingleShopContext and isMultiShopFeatureUsed %}
+            {% include '@PrestaShop/Admin/Improve/Design/Theme/Blocks/multishop_switch.html.twig' %}
+          {% endif %}
+          <div class="card">
+            <div class="card-header">
+              {{ 'Logo'|trans({}, 'Admin.Global') }}
+            </div>
+            <div class="card-body logo-configuration-card-body">
+              {% include '@PrestaShop/Admin/Improve/Design/Theme/Blocks/logo_configuration.html.twig' %}
+            </div>
+            <div class="card-footer">
+              <button class="btn btn-primary float-right">
                 {{ 'Save'|trans({}, 'Admin.Actions') }}
               </button>
+              <div class="clearfix">&nbsp;</div>
             </div>
           </div>
-        </div>
-        {{ form_end(adaptThemeToRtlLanguagesForm) }}
-      </div>
-    </div>
-  {% endif %}
+        {{ form_rest(shopLogosForm.options) }}
+        {{ form_end(shopLogosForm.options) }}
 
-  {% if notUsedThemes is not empty %}
-    <div class="row">
-      <div class="col">
         <div class="card">
-          <h3 class="card-header">
-            <i class="material-icons">settings</i>
-            {{ 'Select a theme for the "%name%" shop'|trans({'%name%': shopName}, 'Admin.Design.Feature') }}
-          </h3>
-          <div class="card-body">
-            {% if not isSingleShopContext %}
-              <div class="alert alert-info mb-0">
-                <p class="alert-text">
-                  {{ 'You must select a shop from the above list if you wish to choose a theme.'|trans({}, 'Admin.Design.Help') }}
-                </p>
-              </div>
-            {% else %}
-              <div class="row">
-                {% for theme in notUsedThemes %}
-                  <div class="col-3">
-                    <div class="card" style="max-width: 350px; max-height: 250px; overflow-y: hidden;">
-                      <h3 class="card-header text-center">
-                        {{ theme.name }}
-                      </h3>
-                      <img src="{{ baseShopUrl }}{{ theme.get('preview') }}" alt="{{ theme.name }}" class="card-img-top">
-                    </div>
+          <div class="card-header">
+            {#todo: new translation#}
+            {{ 'My theme for <<%name%>> shop'|trans({'%name%': shopName}, 'Admin.Design.Feature') }}
```

Comment:
Is the use of square brackets normal here?

Reply:
@LouiseBonnard  they existed in the previous translation and also are visible on zeplin sketch: https://zpl.io/2EPr0PY